packages/multitest/tests/drand-gateway.rs

```diff
@@ -32,10 +34,10 @@ fn integration_test() {
             .unwrap();
     });
 
-    let owner = addr("owner");
-    let bossman = addr("bossman");
-    let manager = addr("manager");
-    let sink = addr("sink");
+    let owner = app.api().addr_make("owner");
+    let bosman = app.api().addr_make("bosman");
```

Comment:
This was supposed to be a "boss man" (fun word for boss that people in The Gambia use often). But whatever, any name works here

Reply:
Ok, nice, now the old name makes more fun, I will revert this to the original ;-)